src/pages/home-page/HomePage.js

```diff
@@ -1,7 +1,7 @@
 import "./home-page.scss";
-import "./../../style/common/common.scss";
-import logo from "../../assets/logo.png";
-import LoginForm from "../../components/users/LoginForm/LoginForm";
+import "style/common/_common.scss";
+import logo from "assets/logo.png";
+import LoginForm from "components/users/LoginForm/LoginForm";
```

Comment:
It's a common practices to follow a structured order for dependencies: 

1. React
2. External libraries dependencies
3. Absolute imports to your own code: E.g. `assets/logo`
4. Relative imports './home/page.scss/'

It's not mandatory follow this pattern but it's a common practice